glmnet/errors.py

```diff
@@ -0,0 +1,43 @@
+import warnings
+
+
+def _check_glmnet_error_flag(jerr, n_lambda):
+    """Check the error flag. Issue warning on convergence errors (jerr < 0)
+    and exception on anything else."""
```

Comment:
Your intro paragraph that describes the interpretation of `jerr` is good -- maybe stash it here in the docstring for a quick reference?